db/migrate/20220218155917_create_roles.rb

```diff
@@ -0,0 +1,10 @@
+class CreateRoles < ActiveRecord::Migration[6.1]
+  def change
+    create_table :roles do |t|
+      t.string :name, null: false 
```

Comment:
Una restricción de unicidad sería recomendable en este campo y en la validación en el modelo. Te dejo una lectura al respecto:
https://thoughtbot.com/blog/the-perils-of-uniqueness-validations